br/pkg/lightning/mydump/loader.go

```diff
@@ -417,6 +418,13 @@ func (s *mdLoaderSetup) route() error {
 		return errors.Trace(err)
 	}
 
+	// check count in knownDBNames should always greater than 0, this will not happen if there are no bugs in the code
+	for dbName, info := range knownDBNames {
+		if info.count < 0 {
+			return common.ErrTableRoute.GenWithStack(fmt.Sprintf("dbName: %s route count less than 0", dbName))
```

Comment:
`GenWithStack` accepts format string and arguments.

BTW,  this loop can be merged into next loop, both of them are reading the count.

Reply:
Could you optimize the error message for user readable?

Reply:
fixed in https://github.com/pingcap/tidb/pull/33484/commits/4c58c60b46fafffc5d20452f70189e1b8befb07d